src/components/VmDialog/index.js

```diff
@@ -128,6 +131,20 @@ class VmDialog extends React.Component {
     return last ? last.message : undefined
   }
 
+  getMemoryPolicy () {
+    const cluster = this.getCluster()
+    const overCommitPercent = cluster && cluster.getIn(['memoryPolicy', 'overCommitPercent'])
+    let guaranteed = overCommitPercent ? (this.state.memory * (100 / overCommitPercent)) : this.state.memory
+
+    const memoryPolicy = {
+      'max': this.state.memory * MAX_VM_MEMORY_FACTOR,
```

Comment:
is this only for next run or also hotplug? If it is for next_run only than it is ok, if it is about hotplug than the "max" should stay as is.